cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -78,6 +79,40 @@ def issue_jwt_token_with_jwk(token_body, file_name, algorithm = Algorithms::RS25
     )
   end
 
+  def issue_jwt_token_with_x5c(token_body, algorithm = Algorithms::RS256)
+    cert = self_signed_certificate(rsa_keys[@default_file_name])
+
+    @jwt_token = JWT.encode(
+      token_body,
+      rsa_keys[@default_file_name],
+      algorithm,
+      {
+        kid:  base64_x5t_from_certificate(cert),
+        x5c:  cert.to_pem.gsub("\n", ""),
+        x5t:  base64_x5t_from_certificate(cert)
+      }
+    )
+  end
+
+  def issue_jwt_token_with_x5u(token_body, file_name, algorithm = Algorithms::RS256)
+    @default_file_name ||= file_name
```

Comment:
I'm a bit lost with default_file_name is it of JWKS or is it of pem endpoint?

Reply:
youre right, `default_file_name` is confusing here, ill get rid of it